src/routes/Home.js

```diff
@@ -167,6 +174,9 @@ export class Home extends Component {
       this.showDeleteLogsSuccessfulMessage(jobExecutionDetails.length);
     };
 
+    // disable all checkboxes while deletion in progress
+    this.setState({ isCheckboxesDisabled: true });
```

Comment:
What if we add state value `isLogsDeletionInProgress`, set it to `true` here, and once the updated logs list is fetched - set it to `false`, such state value naming is more readable, what do you think?

Reply:
@mariia-aloshyna good point.